rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/cluster.stats/30_ccs_stats.yml

```diff
@@ -79,6 +80,10 @@
         body:
           foo: bar
 
+  - do:
+      indices.flush:
+        index: test
```

Comment:
Do the stats count on us flushing? If a new index reader is needed should we refresh instead?

Reply:
Not sure how necessary this is, just in case to ensure bytes really get to disk and are counted.